acceptance/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+CONFIG_FILE=config
+SAMPLE_APP_DIR=sample-android-mixpanel-integration
+TESTS=(test.py)
+NODE_SERVER_PID=NULL
+SELEDROID_PID=NULL
+NPM_PACKAGES_REQUIRED=(express)
+
+VERBOSE=0
+
+function npm_package_check {
+	for package in $NPM_PACKAGES_REQUIRED
+	do
+		ls node_modules 2> /dev/null | grep $package > /dev/null
+		if [ $? != 0 ]
+		then
+			echo "$package is missing. installing.."
+			if [ $VERBOSE != 0 ]
+			then
+				sudo npm install $package --save
+			else
+				sudo npm install $package --save > /dev/null
+			fi
+		fi
+	done
+}
+
+function parse_config {
```

Comment:
Let's see if we can get away with a dummy value here, so you can eliminate the config file and editing Java with sed